scripts/dep_update.sh

```diff
@@ -29,7 +29,7 @@ if [[ "${COMPONENT_DEPENDENCIES_UPDATE}" == 'true' ]]; then
   apt-get update
   apt-get install -y python3-pip
   pip3 install -c "${WORKSPACE}/rpc-gating/constraints_rpc_component.txt" rpc_component
-  component --releases-repo 'https://github.com/mattt416/rpc-metadata' dependency update-requirements
+  component --releases-repo 'https://github.com/rcbops/releases' dependency update-requirements
```

Comment:
That URL is the default, so you can just remove this argument.

Reply:
Done.